src/build.ts

```diff
@@ -504,6 +504,8 @@ async function _watch(nitro: Nitro, rollupConfig: RollupConfig) {
     join(dir, "api"),
     join(dir, "routes"),
     join(dir, "middleware", GLOB_SCAN_PATTERN),
+    join(dir, "plugins"),
```

Comment:
Please split unrelated changes to new PR

Reply:
Yes, you're right! https://github.com/unjs/nitro/pull/1800